components/FormCard.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useState } from 'react'
+import { Button } from './theme/Button'
+import { MdInput } from './MdInput'
+
+// capitalizes first letter of a string
+const capitalizeFirst = (str: string) => {
+  return str.replace(/./, char => char.toUpperCase())
+}
+type Option = {
+  title: string
+  placeHolder?: string
+  value?: string | number
+}
+
+type Btn = {
+  title: string
+  onClick?: Function
+}
+
+type FormCardProps = {
+  values: Option[]
+  buttons?: Btn[]
```

Comment:
what is this option for? could we also have a function as a prop? to be called when users hit submit button. this prop should be required.

Reply:
the function is passed into the buttons prop. This component will be re-done using formik

Reply:
the function prop is the `button` prop. i will change the naming.